src/include/86box/video.h

```diff
@@ -225,30 +225,32 @@ extern int  video_card_get_flags(int card);
 extern int 	video_is_mda(void);
 extern int 	video_is_cga(void);
 extern int 	video_is_ega_vga(void);
-extern void	video_inform(int type, const video_timings_t *ptr);
+#define video_inform(type, video_timings_ptr) video_inform_monitor(type, video_timings_ptr, monitor_index_global)
```

Comment:
Should probably move all the defines together to keep the header tidy.

Reply:
Done.